source/configuration_manual/mail_location/obox/dictmap.rst

```diff
@@ -184,6 +184,19 @@ Dictmap Parameters
 |                                 |                                                                              |
 |                                 | .. versionadded:: v2.3.10                                                    |
 +---------------------------------+------------------------------------------------------------------------------+
+| cleanup-uncertain               | If this setting is enabled, and a write to Cassandra has been uncertain like |
```

Comment:
I think this is somewhat cumbersome sentence altogether. Perhaps rewrite with less commas and more dots?

Reply:
Yes it did read horribly, now it should be better.